components/MainDisplay.jsx

```diff
@@ -1,31 +1,38 @@
 import { Sidebar } from "./Sidebar";
 
-export const MainDisplay = () => {
-
-    return (
-        <section className="mainLayout h-screen bg-gray-100">
-          <div className="max-w-full h-full pb-4 ml-12 mr-12 grid grid-cols-12">
-            <div className="col-span-2 h-full bg-gray-100 text-black">
-              <div className="courseNav container mt-4 h-1/4">
-                <div>
-                  <h1 className="mx-auto font-bold text-black border-b-2"> Courses </h1>
-                </div>
-                <div className="mt-2">
-                  <Sidebar></Sidebar>
-                </div>
-              </div>
-              <div className="workNav container h-3/4">
-                <h1 className="mx-auto font-bold text-black border-b-2">
-                  Modules
-                </h1>
-              </div>
+export const MainDisplay = (claims) => {
+  if (claims.claims.admin) {
+    console.log("is an admin!")
```

Comment:
we can see in console if a user is a admin or student though the claims prop passed in through index.js